distribution/lib/Standard/Base/0.0.0-dev/src/Function.enso

```diff
@@ -7,3 +9,56 @@ type Function
        the this argument.
     @Builtin_Type
     type Function
+
+
+## An identity function which returns the provided argument.
```

Comment:
At least as it currently stands we should make these methods:

```python
    ## An identity function which returns the provided argument.
    
       Arguments:
       - x: the value to return.
    
       > Example
         five = Function.identity 5 # returns number 5
    
    identity : a -> a
    identity x = x
```

But should they be global static functions? Not that I am sure we can do those yet?

Reply:
I've changed the functions definition in [0a8cdbe](https://github.com/enso-org/enso/pull/3554/commits/0a8cdbe2963a7d792ea00e605feb3b49a603ed00), but I still have to reference them as `Function.identity` in the test. Need a hint of what else to do from language experts.

Reply:
Need an indent of `    ` to make these member functions.

Then can remove `import Standard.Base.Function` from the test script.

I am not sure naked static function calls are possible yet.

Reply:
This is now done